setup.cfg

```diff
@@ -36,6 +36,68 @@ install_requires =
     decorator
     prometheus_client
 
+[flake8]
+# max line length for black
+max-line-length = 88
```

Comment:
Can we keep this at 120? That will match the check in the CI, and is what we've historically used, so better matches the existing codebase.